src/main/java/net/dancier/dancer/dancers/DancerService.java

```diff
@@ -36,4 +39,23 @@ public HashMap<UUID, DancerDto> getDancerMap(DancerIdsDto dancerIdsDto) {
 
         return dancers;
     }
+
+    public List<PublicProfileDto> getDancersList(AuthenticatedUser authenticatedUser, Gender gender, int range) {
+
+        Dancer dancer = loadByUserId(authenticatedUser.getUserId());
+        Double longitudeRange = (double)range/112;
```

Comment:
Das könnte ein Bug sein aber ich bin mir nicht sicher, die Division hier `range/122` müsste eine Integer-Division sein, die dann als Output in ein Double gewandelt wird. Aber auch wenn das nicht der Fall ist würde ich das von der Konsistenz her eher so schreiben, wie in der Zeile darunter:
```suggestion
        Double longitudeRange = range/112.0;
```